src/pages/workspaces/workspace/Data.js

```diff
@@ -545,9 +552,13 @@ const EntitiesContent = ajaxCaller(class EntitiesContent extends Component {
                         ])
                       ]),
                       cellRenderer: ({ rowIndex }) => {
-                        return renderDataCell(
-                          Utils.entityAttributeText(entities[rowIndex].attributes[name]), namespace
-                        )
+                        const dataInfo = entities[rowIndex].attributes[name]
+                        return h(Fragment, [
+                          (dataInfo && _.isArray(dataInfo.items)) ? linkButton({
```

Comment:
Nit: coerce `dataInfo` to a boolean, I think.

Are there any values that `dataInfo` can have that would lead to the wrong behavior here?

Reply:
Oh wait, do you mean like do a: `!!dataInfo`?

Reply:
`dataInfo` effectively comes from users. I think it's probably okay in this case. We only treat it differently if it's an array which will always be truthy. Anything else, whether truthy or falsey, is treated the same, so user-supplied `0` or `""` won't cause any problems. I just want to put the idea in other people's heads in case I missed a case.